assets/javascripts/modules/mma/updatePayment.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
```

Comment:
There are a few consolo.logs in this file

Reply:
😞 removed

Reply:
👍